src/Entity/Content.php

```diff
@@ -450,12 +450,18 @@ public function removeTaxonomy(Taxonomy $taxonomy): self
      * - {{ record.image }} => field named image
      * - {{ record|image }} => value of guessed image field
      */
-    public function __call(string $name, array $arguments = []): Field
+    public function __call(string $name, array $arguments = [])
     {
         try {
-            return $this->getField($name);
+            $field = $this->getField($name);
         } catch (\InvalidArgumentException $e) {
             throw new \RuntimeException(sprintf('Invalid field name or method call on %s: %s', $this->__toString(), $name));
         }
+
+        if ($field->getDefinition()->get('allow_html')) {
+            $field = new \Twig_Markup($field, 'UTF-8');
+        }
+
+        return $field;
```

Comment:
can we assume that magic __call will be used only in Twig and return $field->getTwigValue() here?
Then we can move checking for allow_html there.

Reply:
Yes, I think so.. Will look into that!